### ISSUE TITLE
Feature request: Run a job once

### ISSUE THREAD

**Issue by JohnRoesler** [go-co-op/gocron#24]
From @cybervagabond

Maybe it will be good idea.
In some cases people neen to run task only once.
Yes i can run task and then remove.

**Comment by Streppel** [comment]
This can be easily achieved with [AfterFunc](https://golang.org/pkg/time/#AfterFunc), no need to import a third party package. I believe this is out of scope as it is unrelated to a cron execution model.

**Comment by excxapp** [comment]
This means delay job, I think we need cron and delay job(run one time). like timewheel,this package only have period job,I think have delay job(one time).

**Comment by brunovu20** [comment]
I have the same usecase and I do use AfterFunc of time, for example:

```
job, err := s1.Every(3).Seconds().Do(task)
if err == nil {
	time.AfterFunc(time.Duration(5*time.Second), func() {
		fmt.Println("stop job now!")
		s1.RemoveByReference(job)
	})
}
```

Hope that helps

**Comment by Streppel** [comment]
Thanks for the feedback. We'll be adding this to our v1 release as this has been being requested by lots of people :cool:

**Comment by arjunmahishi** [comment]
@Streppel @JohnRoesler 
Instead of restricting this to run "once", we could make it a little more flexible. What if we had something like this:
```go
    s := gocron.NewScheduler(time.UTC)
    job1, _ := s.Every(3).Seconds().Do(task1)
    job1.LimitRunsTo(5)
    job2, _ := s.Every(2).Seconds().Do(task2)
    job2.LimitRunsTo(1)
    s.StartAsync()
```
